base/sysinfo.jl

```diff
@@ -141,12 +141,22 @@ function cpu_info()
     return cpus
 end
 
+"""
+    uptime()
+
+Gets the current system uptime.
```

Comment:
if they're unexported then you need the prefix or an import to call them. one of the guidelines for when something unexported is supposed to be used publicly is if it's documented (incl the prefix if that's the usual usage, like Profile.print)